cookbooks/aws-parallelcluster-tests/recipes/docker_mock.rb

```diff
@@ -0,0 +1,85 @@
+return unless on_docker?
+
+file '/bin/systemctl' do
+  content %{
+#!/bin/bash
+
+if [ "$1" == "--system" ]; then
```

Comment:
What is the effect of this option?
I would have expected the setting of a variable to change some behaviour...

Reply:
I guess this part is not used anymore. I will create a task to cleanup this script which is a mere copy/paste from another cookbook.